### DISCUSSION TITLE
Flows Conditions

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15556] [Archive]
Hi,

I have built a simple Flow to understand how Flow conditions works and this flow doesn't work correctly.,. What have I'm doing wrong? 

1. Trigger: for collection Articles Update
![conditions flow](https://user-images.githubusercontent.com/54798086/189947337-218fc79a-24d6-4d28-8dbf-d6de33ee449c.PNG)

2. Read Data: with ID from updated Article from Trigger 
![conditions1](https://user-images.githubusercontent.com/54798086/189947567-f12a65cf-495a-4f36-a366-c3d6867de4df.PNG)

3. Condition: if "article_type" equal "tweet" -> send notification MATCH. If NOT equal "tweet" -> send notification NOT MATCH.
![conditions3](https://user-images.githubusercontent.com/54798086/189948338-bd0c3323-29c4-4b38-ab13-24232a050df5.PNG)

**Comment by licitdev** [comment]
@rynskiy Your condition looks correct. The issue seems to be in the Read Data operation. Please try to set the ID to be `{{$trigger.keys[0]}}`, without the query.

![image](https://user-images.githubusercontent.com/26413686/189966039-e0bf0c07-2c85-40fa-b21c-ed22b2115923.png)
